### PR TITLE
feat: Updated all Swedish Companion translations, minor changes to radio translations 🇸🇪

### PR DESCRIPTION
**Pull request by ulfhedlund**
Summary of changes:
- Regenerated the existing Swedish translation file for Companion using QT Linguist lupdate tool, then added all translations still missing. As of now, this translation is complete.
- Added translation in code for a few missing strings from #2148 
- Minor changes to se.h radio translations.

### CODE REVIEW

File: radio/src/translations/se.h [hunk 1]
```diff
@@ -655,33 +674,38 @@
...
+#define TR_RENAME_LABEL                 "Byt namn på etikett"
+#define TR_DELETE_LABEL                 "Radera etikett"
 #define TR_DUPLICATE_MODEL              "Duplicera modell"
 #define TR_COPY_MODEL                   "Kopiera modell"
 #define TR_MOVE_MODEL                   "Flytta modell"
 #define TR_BACKUP_MODEL                 "Modellbackup"
 #define TR_DELETE_MODEL                 "Ta bort modell"
 #define TR_RESTORE_MODEL                "Återställ modell"
 #define TR_DELETE_ERROR                 "Delete error"
+#define TR_CAT_NOT_EMPTY                "Kategorin är ej tom"
```
Comment (pfeerick) [inline radio/src/translations/se.h:702]: It looks like it's for deleting a non-empty category so can also go

File: radio/src/translations/se.h [hunk 2]
```diff
@@ -1142,15 +1170,21 @@
...
 #define TR_REMOVE_WIDGET                "Ta bort widget"
 #define TR_WIDGET_SETTINGS              "Widgetinställningar"
 #define TR_REMOVE_SCREEN                "Ta bort huvudvy"
-#define TR_SETUP_WIDGETS                "Konfigurera widgets"
+
+#if LCD_W > LCD_H
+  #define TR_SETUP_WIDGETS              "Konfigurera widgets"
+#else
+  #define TR_SETUP_WIDGETS              "Konfig. widgets"
+#endif
```
Comment (pfeerick) [inline radio/src/translations/se.h:1186]: If you choose TR(short,long), this once also needs updating for consistency

**Review by pfeerick — CHANGES_REQUESTED** [review]
Firstly, thank you very much for this! :) I may also needpick your brain over the QT Linquist updatey thing, since it looks like I need to start running that every so often so things on the Companion translation side keep up to date. 😆 

Just some minor tweaks on the radio firmware side needed - the companion changes seem fine, and I'm going to assume the `.ts` file is fine since that was automatically processed and survived compilation ;)

File: radio/src/translations/se.h [hunk 3]
```diff
@@ -270,6 +279,12 @@
 
 #define TR_ON_ONE_SWITCHES              "PÅ","Ett"
 
+#if defined(GYRO)
+  #define TR_GYR_VSRCRAW                "GyrX","GyrY"
+#else
+  #define TR_GYR_VSRCRAW
+#endif
+
```
[the remark this answers is not in the document]
Comment (ulfhedlund) [reply radio/src/translations/se.h:287]: Gone!

File: radio/src/translations/se.h [hunk 4]
```diff
@@ -655,33 +674,38 @@
 #define TR_VOLTAGE                      INDENT"Volt"
 #define TR_CURRENT                      INDENT"Ampere"
 #define TR_SELECT_MODEL                 "Välj modell"
+#define TR_MODEL_CATEGORIES             "Modellkategori"
 #define TR_MODELS                       "Modeller"
-#define TR_SELECT_MODE                  "Select mode"
+#define TR_SELECT_MODE                  "Välj läge"
+#define TR_CREATE_CATEGORY              "Skapa kategori"
+#define TR_RENAME_CATEGORY              "Byt kategorinamn"
+#define TR_DELETE_CATEGORY              "Ta bort kategori"
```
[the remark this answers is not in the document]
Comment (ulfhedlund) [reply radio/src/translations/se.h:682]: Gone!

File: radio/src/translations/se.h [hunk 5]
```diff
@@ -155,20 +160,19 @@
...
 #endif
 
 #define TR_SF_SCREENSHOT                "Skärmbild"
 #define TR_SF_RACING_MODE               "Tävlingsläge"
 #define TR_SF_DISABLE_TOUCH             "Ej pekskärm"
-#define TR_SF_SET_SCREEN                "Set Main Screen"
 #define TR_SF_RESERVE                   "[reserv]"
 
-#define TR_VFSWFUNC                     TR_SF_SAFETY,"Lärare","Sätt trimm","Nollställ","Sätt",TR_ADJUST_GVAR,"Volym","Sätt failsafe","Range check","Module bind",TR_SOUND,TR_PLAY_TRACK,TR_PLAY_VALUE, TR_SF_RESERVE,TR_SF_PLAY_SCRIPT,TR_SF_RESERVE,TR_SF_BG_MUSIC,TR_VVARIO,TR_HAPTIC,TR_SDCLOGS,"Belysning", TR_SF_SCREENSHOT,TR_SF_RACING_MODE,TR_SF_DISABLE_TOUCH,TR_SF_SET_SCREEN TR_SF_TEST
+#define TR_VFSWFUNC                     TR_SF_SAFETY,"Lärare","Spara trimmar","Nollställ","Sätt ",TR_ADJUST_GVAR,"Volym","Sätt failsafe","Range check","Module bind",TR_SOUND,TR_PLAY_TRACK,TR_PLAY_VALUE, TR_SF_RESERVE,TR_SF_PLAY_SCRIPT,TR_SF_RESERVE,TR_SF_BG_MUSIC,TR_VVARIO,TR_HAPTIC,TR_SDCLOGS,"Belysning", TR_SF_SCREENSHOT,TR_SF_RACING_MODE,TR_SF_DISABLE_TOUCH,TR_SF_TEST
```
[the remark this answers is not in the document]
Comment (ulfhedlund) [reply radio/src/translations/se.h:175]: Fixed

File: radio/src/translations/se.h [hunk 6]
```diff
@@ -35,19 +34,25 @@
...
 #define TR_TRNMODE                      "Av","+=",":="
 #define TR_TRNCHN                       "KA1","KA2","KA3","KA4"
 #define TR_AUX_SERIAL_MODES             "AV","Speglad telemetri","Telemetri in","SBUS Lärare","LUA","CLI","GPS","Debug"
-#define TR_SWTYPES                      "Ingen",TR("Flipp","2 pos momentan"),"2 pos","3 pos"
+
+#if LCD_W > LCD_H
+  #define TR_SWTYPES                      "Ingen", "2 pos flipp","2 pos","3 pos"
+#else
+  #define TR_SWTYPES                      "Ingen","Flipp","2 pos","3 pos"
+#endif
```
[the remark this answers is not in the document]
Comment (ulfhedlund) [reply radio/src/translations/se.h:48]: Yes, I am aware of that but in this case I couldn't get it right for NV14 using TR macro. I tried.

File: radio/src/translations/se.h [hunk 7]
```diff
@@ -1,8 +1,7 @@
 /*
- * Copyright (C) EdgeTX
+ * Copyright (C) OpenTX
  *
  * Based on code named
- *   opentx - https://github.com/opentx/opentx
  *   th9x - http://code.google.com/p/th9x
```
[the remark this answers is not in the document]
Comment (ulfhedlund) [reply radio/src/translations/se.h:5]: I'm not aware of changing it so if I did it was unintentional. Now same as in en.h.

File: radio/src/translations/se.h (hunk 1 above)
Comment (ulfhedlund) [reply radio/src/translations/se.h:702]: Gone!

File: radio/src/translations/se.h (hunk 2 above)
Comment (ulfhedlund) [reply radio/src/translations/se.h:1186]: I'll leave it as is, see above reason.

**Comment by ulfhedlund** [comment]
@pfeerick: I am not quite sure what to do with your review (still very new to github etc) but I have updated the se.h file as per your instructions. Let me know if I need to do anything else.

**Comment by pfeerick** [comment]
You did good ;) Basically just resolve as you go, and comment otherwise if there is more to it... i.e. like the NV14. All good. I'll look at this properly either tomorrow or Sunday... club AGM tomorrow so more likely Sunday.

**Comment by ulfhedlund** [comment]
Great 👍

**Comment by ulfhedlund** [comment]
So lupdate can be used to continously keep the Qt .ts files updated  but the firmware .h files have no similar tool. 
Sometimes I have discovered missing items when building firmware through error messages and then added them. But sometimes developers have added items to each of the .h files and then it is a bit difficult to know what has been added and needs translation. I've tried Github File Watcher to get notified when the se.h file is altered but can't get that to work. 

Anything that could help the translators would be very useful. :)

**Comment by pfeerick** [comment]
> So lupdate can be used to continously keep the Qt .ts files updated but the firmware .h files have no similar tool. Sometimes I have discovered missing items when building firmware through error messages and then added them. But sometimes developers have added items to each of the .h files and then it is a bit difficult to know what has been added and needs translation. I've tried Github File Watcher to get notified when the se.h file is altered but can't get that to work.
> 
> Anything that could help the translators would be very useful. :)

Indeed. That is part of a larger project I'm hoping to work out soon - using https://crowdin.com to manage the translation side of things. Just not enough hours in the day of late to spend time looking into how to integrate that into current process.

**Comment by ulfhedlund** [comment]
Thanks, I'll try that!

**Review by pfeerick — APPROVED** [review]
Looks good now. I just had to add a define back as I didn't notice it missing before. You should also see a commit with some couple more strings being available for translation on the radio side in the next day or so also... ;)

**Comment by ulfhedlund** [comment]
I have made one in the .ts-file, if it isn't too late. Just a very minor mistake I made.

**Comment by ulfhedlund** [comment]
Just saw you already merged this, so I'll add it to another PR.
Thanks you for all your help!

**Pull request merged**